src/webWorker/webWorkerManager.js

```diff
@@ -115,9 +129,25 @@ function spawnWebWorker () {
   if (webWorkers.length >= config.maxWebWorkers) {
     return
   }
+  const { PUBLIC_URL } = window
+
+  let workerUrl
+  if (!PUBLIC_URL) {
```

Comment:
GRRR - damn webpack isn't doing what it is documented to do, so I'm needing to try to figure out work arounds.
The line 140 below is SUPPOSED to be the right way to do things, but it never works.